### ISSUE TITLE
A new tab opens evertime I start VSCode

### ISSUE THREAD

**Issue by oasaleh** [brainomite/dark-synthwave-vscode#7]
Besides the fact that the Glow feature doesn't work as the command does not exist, there seems to be another issue. Everytime I start VSCode, a new tab titled "What's new for Synthwave '84" opens. I really like the extention, but I can't use it like this. I'll keep an eye for an update.

Thank you.
![image](https://user-images.githubusercontent.com/56449814/171074068-89ee4c2d-1f8d-4cba-8fe3-f7e85aaf745d.png)
![image](https://user-images.githubusercontent.com/56449814/171074092-e31597f8-1c50-49eb-ba15-17b5f533e80d.png)

**Comment by mazwrld** [comment]
Same

**Comment by iandouglas** [comment]
I opened the "src/extension.js" for this extension and added a "return" statement at the top, and the problem went away:

```
function showUpdatePage() {
    return; // add this line

		const panel = vscode.window.createWebviewPanel(
			`synthwave.whatsNew`, // Identifies the type of the webview. Used internally
			'What\'s new for Synthwave \'84', // Title of the panel displayed to the user
			vscode.ViewColumn.One, // Editor column to show the new webview panel in.
			{ enableScripts: !0 } // Webview options. More on these later.
		);

		const viewPath = path.join(this.cntx.extensionPath, "whats-new", "view.html");
		const viewResourcePath = panel.webview.asWebviewUri(viewPath);
		const htmlContent = fs.readFileSync(viewPath, "utf-8");
		panel.webview.html = htmlContent;
}
```

**Comment by brainomite** [comment]
I removed the tab altogether - sorry about that this comment closes #7
